src/v/cloud_storage/tests/partition_manifest_test.cc

```diff
@@ -1883,3 +1883,59 @@ SEASTAR_THREAD_TEST_CASE(test_generate_segment_name_format) {
         BOOST_REQUIRE_EQUAL(expected, actual2);
     }
 }
+
+SEASTAR_THREAD_TEST_CASE(test_cloud_log_size_updates) {
```

Comment:
Could use a test for serializing and deserializing? And a test that even when deserializing the old format we get a size

Reply:
> Could use a test for serializing and deserializing?

One already exists in `partition_manifest_test.cc`: `test_manifest_serialization`.

> And a test that even when deserializing the old format we get a size

Nice idea. Added one.